bg96_gps.device.lib.nut

```diff
@@ -390,14 +390,19 @@ BG96_GPS <- {
 
         if (_session != null) {
             local t = _session.assist.read();
-            local valid = (t.status == 0);
-            local data = {};
-            data.time <- _getValidTime(t.injecteddatatime,t.xtradatadurtime);
-            valid = data.time > 0
-            data.valid <- valid;
-            local msg = valid ? "GNSS assist data valid" : "GNSS assist data invalid";
-            _notify(msg, data);
-            return valid;
+            if (t.status == 0) {
+                local valid = (t.status == 0);
```

Comment:
this always gets overwritten, we can probably remove the local variable and just do `data.valid <- data.time > 0;` and then refer to `data.valid` in the ternary test and return value